topi/python/topi/cpp.py

```diff
@@ -0,0 +1,4 @@
+"""FFI for C++ TOPI ops and schedules"""
+from ._ffi.cpp_init import _init_api
+
```

Comment:
it is likely we only need to optionally load libtopi if it can be found and run the cpp init. Let us not require libtvm_topi for now, and just use it for test purpose. Once logic transfer to c++ complete, we can mark it as required

Reply:
Ok so you're meaning add tests, but don't add them to the CI build yet?

Reply:
And should I reset the jenkinsfile to the current master?

Reply:
we do need to add them to topi tests. Just not require user to do this. I am not sure what is happening on the Jenkinsfile side and why things didn't get packed

Reply:
As a temporary solution, let us build libtopi_tvm at the cpp test phase and do the test there

Reply:
ok, would that be running the python test of the exported ops/schedules in the cpp test phase?

Reply:
yes